Applications/spire/source/time_and_sales/time_and_sales_properties_dialog.cpp

```diff
@@ -448,3 +470,13 @@ void time_and_sales_properties_dialog::update_colors(int band_index) {
     .arg(m_properties.get_text_color(i).name());
   m_band_list->setStyleSheet(m_band_list_stylesheet + selected_stylesheet);
 }
+
+void time_and_sales_properties_dialog::update_font_preview_stylesheet() {
+  auto font_metrics = QFontMetrics(m_properties.m_font);
+  auto font_height = font_metrics.size(0, "Test").height();
```

Comment:
Instead of "Test" use "EM". An "EM" block is the standard used to measure the maximum width and height of a font.